--- meson.build_org     2020-04-29 12:10:01.843127148 +0000
+++ meson.build 2020-04-29 12:10:11.809986299 +0000
@@ -24,7 +24,7 @@
 dependency('gtk+-3.0', version: '>= 3.22')
 dependency('libsoup-2.4')
 dependency('glib-2.0')
-dependency('pygobject-3.0', version: '>= 3.29.1')
+dependency('pygobject-3.0', version: '>= 3.28.0')

 python_dir = join_paths(get_option('prefix'), python.sysconfig_path('purelib'))
 LIBEXEC_DIR = join_paths(get_option('prefix'), get_option('libexecdir'))
