diff -Naur libretro-super-Latest.org/libretro-build-common.sh libretro-super-Latest/libretro-build-common.sh
--- libretro-super-Latest.org/libretro-build-common.sh	2021-07-09 12:13:55.665221149 +0000
+++ libretro-super-Latest/libretro-build-common.sh	2021-07-09 12:14:57.117694502 +0000
@@ -378,6 +378,8 @@
 		secho "$(color 31)$num_fail core(s)$(color) failed:"
 		lecho "$num_fail core(s) failed:"
 		lsecho "$fmt_fail"
+		# Ensure that build fails for container debugging
+		exit 1
 	fi
 }
 
diff -Naur libretro-super-Latest.org/libretro-build.sh libretro-super-Latest/libretro-build.sh
--- libretro-super-Latest.org/libretro-build.sh	2021-07-09 12:13:55.668221171 +0000
+++ libretro-super-Latest/libretro-build.sh	2021-07-09 12:15:21.692885105 +0000
@@ -273,8 +273,8 @@
 				libretro_build_core bnes
 			fi
 		fi
-
-		build_libretro_test
+		# Testing currently fails
+		#build_libretro_test
 	fi
 }
 
diff -Naur libretro-super-Latest.org/libretro-config.sh libretro-super-Latest/libretro-config.sh
--- libretro-super-Latest.org/libretro-config.sh	2021-07-09 12:13:55.669221179 +0000
+++ libretro-super-Latest/libretro-config.sh	2021-07-09 12:14:18.454394631 +0000
@@ -142,7 +142,7 @@
 #LIBRETRO_LOG_CORE="%s.log"
 
 # Comment this if you don't need to see developer output
-LIBRETRO_DEVELOPER=1
+#LIBRETRO_DEVELOPER=1
 
 
 # BUILD_REVISIONS
