--- engines/mohawk/myst_areas.cpp_org	2020-11-29 08:53:59.365509021 +0100
+++ engines/mohawk/myst_areas.cpp	2020-11-29 08:54:40.650965609 +0100
@@ -189,7 +189,9 @@
 		_videoFile += c;
 	} while (c);
 
-	rlstStream->skip(_videoFile.size() & 1);
+	if (!(_videoFile.size() & 1)) {
+		rlstStream->skip(1);
+	}
 
 	// Trim method does not remove extra trailing nulls
 	while (_videoFile.size() != 0 && _videoFile.lastChar() == 0)
