--- engines/sci/sci.cpp.org	2019-06-03 18:47:18.676080160 +0200
+++ engines/sci/sci.cpp	2019-06-03 18:47:54.127610570 +0200
@@ -381,16 +381,16 @@
 
 	// Show any special warnings for buggy scripts with severe game bugs,
 	// which have been patched by Sierra
-	if (getGameId() == GID_KQ7 && ConfMan.getBool("subtitles")) {
-		showScummVMDialog(_("Subtitles are enabled, but subtitling in King's"
-						  " Quest 7 was unfinished and disabled in the release"
-						  " version of the game. ScummVM allows the subtitles"
-						  " to be re-enabled, but because they were removed from"
-						  " the original game, they do not always render"
-						  " properly or reflect the actual game speech."
-						  " This is not a ScummVM bug -- it is a problem with"
-						  " the game's assets."));
-	}
+	//if (getGameId() == GID_KQ7 && ConfMan.getBool("subtitles")) {
+	//	showScummVMDialog(_("Subtitles are enabled, but subtitling in King's"
+	//					  " Quest 7 was unfinished and disabled in the release"
+	//					  " version of the game. ScummVM allows the subtitles"
+	//					  " to be re-enabled, but because they were removed from"
+	//					  " the original game, they do not always render"
+	//					  " properly or reflect the actual game speech."
+	//					  " This is not a ScummVM bug -- it is a problem with"
+	//					  " the game's assets."));
+	//}
 
 	// Show a warning if the user has selected a General MIDI device, no GM patch exists
 	// (i.e. patch 4) and the game is one of the known 8 SCI1 games that Sierra has provided
